.github/workflows/perf.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           export SWIFTENV_ROOT="$HOME/.swiftenv"
           export PATH="$SWIFTENV_ROOT/bin:$PATH"
           eval "$(swiftenv init -)"
+          ./scripts/install-toolchain.sh
           swiftenv install $TOOLCHAIN_DOWNLOAD
```

Comment:
We need to remove swiftenv install line after merging this PR because older version toolchain is necessary to compare with base branch